queuelib/tests/test_queue.py

```diff
@@ -20,7 +20,7 @@
 
 class DummyQueue:
     def __init__(self) -> None:
-        self.q = list()  # type: list
+        self.q = []  # type: list
```

Comment:
```suggestion
        self.q: list = []
```
This is possible after dropping support for py35

Reply:
Done. 💪🏻